basil/HL/JtagMaster.py

```diff
@@ -93,17 +93,21 @@ def get_word_count(self):
 
     def set_command(self, value):
         """
-        Operation identifier
         IR_SCAN or DR_SCAN
         """
-        self.OPERATION = value
+        if value == "INSTRUCTION":
+            self.OPERATION = self.jtag_command["INSTRUCTION"]
+        elif value == "DATA":
+            self.OPERATION = self.jtag_command["DATA"]
```

Comment:
`self.OPERATION = self.jtag_command[value]` ?

Reply:
You're right, that would make more sense